superset/utils/dashboard_import_export.py

```diff
@@ -38,12 +38,17 @@ def import_dashboards(session, data_stream, import_time=None):
     session.commit()
 
 
-def export_dashboards(session):
+def export_dashboards(session, dashboard_id=None, dashboard_title=None):
     """Returns all dashboards metadata as a json dump"""
     logging.info('Starting export')
     dashboards = session.query(Dashboard)
     dashboard_ids = []
     for dashboard in dashboards:
-        dashboard_ids.append(dashboard.id)
-    data = Dashboard.export_dashboards(dashboard_ids)
+        if dashboard_id or dashboard_title:
```

Comment:
In envs with lots of dashboards, it'd be more efficient to add a query filter, as in:

[untested]
```python
query = session.query(Dashboard)
if dashboard_ids:
    query = query.filter(Dasbhoard.id.in_(dashboard_ids))
if dashboard_titles:
    query = query.filter(Dasbhoard.title.in_(dashboard_titles))
```

Reply:
Good idea.

Reply:
Ended up doing an OR between them in a single query.